src/test/TestGhoStewardsForkRemote.t.sol

```diff
@@ -0,0 +1,338 @@
+import {IGhoAaveSteward} from '../contracts/misc/interfaces/IGhoAaveSteward.sol';
+import {GhoAaveSteward} from '../contracts/misc/GhoAaveSteward.sol';
+import {GhoBucketSteward} from '../contracts/misc/GhoBucketSteward.sol';
+import {GhoCcipSteward} from '../contracts/misc/GhoCcipSteward.sol';
+import {RateLimiter, IUpgradeableLockReleaseTokenPool} from '../contracts/misc/dependencies/Ccip.sol';
+import {IDefaultInterestRateStrategyV2} from '../contracts/misc/dependencies/AaveV3-1.sol';
+import {MockUpgradeableBurnMintTokenPool} from './mocks/MockUpgradeableBurnMintTokenPool.sol';
+
+contract TestGhoStewardsForkRemote is Test {
+  address public OWNER = makeAddr('OWNER');
+  address public RISK_COUNCIL = makeAddr('RISK_COUNCIL');
+  IPoolDataProvider public POOL_DATA_PROVIDER = AaveV3Arbitrum.AAVE_PROTOCOL_DATA_PROVIDER;
+  IPoolAddressesProvider public POOL_ADDRESSES_PROVIDER = AaveV3Arbitrum.POOL_ADDRESSES_PROVIDER;
+  address public GHO_TOKEN = 0x7dfF72693f6A4149b17e7C6314655f6A9F7c8B33;
+  address public GHO_ATOKEN = 0xeBe517846d0F36eCEd99C735cbF6131e1fEB775D;
+  address public ARM_PROXY = 0xC311a21e6fEf769344EB1515588B9d535662a145;
+  address public ACL_ADMIN = AaveV3Arbitrum.ACL_ADMIN;
+  address public GHO_TOKEN_POOL = MiscArbitrum.GHO_CCIP_TOKEN_POOL;
+  address public PROXY_ADMIN = MiscArbitrum.PROXY_ADMIN;
+  address public ACL_MANAGER;
+
+  GhoAaveSteward public GHO_AAVE_STEWARD;
+  GhoBucketSteward public GHO_BUCKET_STEWARD;
+  GhoCcipSteward public GHO_CCIP_STEWARD;
+
+  uint64 public remoteChainSelector = 5009297550715157269;
+
+  event ChainConfigured(
+    uint64 remoteChainSelector,
+    RateLimiter.Config outboundRateLimiterConfig,
+    RateLimiter.Config inboundRateLimiterConfig
+  );
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('arbitrum'), 247477524);
+    vm.startPrank(ACL_ADMIN);
+    ACL_MANAGER = POOL_ADDRESSES_PROVIDER.getACLManager();
+
+    IGhoAaveSteward.BorrowRateConfig memory defaultBorrowRateConfig = IGhoAaveSteward
+      .BorrowRateConfig({
+        optimalUsageRatioMaxChange: 10_00,
+        baseVariableBorrowRateMaxChange: 5_00,
+        variableRateSlope1MaxChange: 10_00,
+        variableRateSlope2MaxChange: 60_00
+      });
+
+    GHO_AAVE_STEWARD = new GhoAaveSteward(
+      OWNER,
+      address(POOL_ADDRESSES_PROVIDER),
+      address(POOL_DATA_PROVIDER),
+      GHO_TOKEN,
+      RISK_COUNCIL,
+      defaultBorrowRateConfig
+    );
+    IAccessControl(ACL_MANAGER).grantRole(
+      IACLManager(ACL_MANAGER).RISK_ADMIN_ROLE(),
+      address(GHO_AAVE_STEWARD)
+    );
+
+    GHO_BUCKET_STEWARD = new GhoBucketSteward(OWNER, GHO_TOKEN, RISK_COUNCIL);
+    GhoToken(GHO_TOKEN).grantRole(
+      GhoToken(GHO_TOKEN).BUCKET_MANAGER_ROLE(),
+      address(GHO_BUCKET_STEWARD)
+    );
+
+    GHO_CCIP_STEWARD = new GhoCcipSteward(GHO_TOKEN, GHO_TOKEN_POOL, RISK_COUNCIL, true);
+
+    address[] memory controlledFacilitators = new address[](1);
+    controlledFacilitators[0] = address(GHO_ATOKEN);
```

Comment:
GHO ATOKEN is not a facilitator in remote deployments

Reply:
Done